stock_deposit/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=============
+Stock deposit
+=============
+
+This module extends standard WMS to manage customers deposit locations by
+warehouse
+
+Installation
+============
+
+The installation module process create a deposit location by each warehouse
+and two picking type, one for out deposits and other for return deposits.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Inventory > Settings > Product* Owners and click
+   "**Manage consignee stocks (advanced)**".
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to products and create one of product type.
+#. Update quantities on hand to have stock of it.
+#. Go to inventory dashboard and click on deposit out card to do a new
+   transfer.
+#. Create a picking with owner and select the product to do the transfer.
+
+After that process you can view de deposit quantities in product form view,
+also in Inventory --> Reports menu the entry "Deposit location inventory"
```

Comment:
Start a new point with `#.` for this and put `You can also see the deposits for all your products in the menu entry Inventory --> Reports --> Deposit location inventory`